app/main.py

```diff
@@ -2,4 +2,70 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(
+            self,
+            name: str,
+            weight: int,
+            coords: list[int] = None
+    ) -> None:
+        self.name = name
+        self.weight = weight
+        self.coords = coords
+        if not coords:
+            self.coords = [0, 0]
```

Comment:
You can simpify your code and write it in one line
```suggestion
        self.coords = coords or [0, 0]
```